contracts/WildcardSteward_v2.sol

```diff
@@ -140,6 +145,9 @@ contract WildcardSteward_v2 is Initializable {
         address oldBenfactor = benefactors[tokenId];
         benefactors[tokenId] = _newReceivingBenefactor;
         benefactorFunds[_newReceivingBenefactor] = benefactorFunds[oldBenfactor];
+        // Maybe we shouldn't also swap over the from old to new benefactor?
```

Comment:
I think lets remove this comment, if it is worth considering we should make a github issue about it.

My stance is that the benefactor can be another smart contract, and then we can implement all of that logic there.